Dockerfile-dev

```diff
@@ -1,4 +1,4 @@
-FROM node:18-alpine
+FROM node:18.15.0-alpine3.17
```

Comment:
🟠 `docker-compose.yml` in root should be updated with:


``` 
   ports:
      - '${HOST_APP_PORT}:3000'
      - '${HOST_PRIVATE_APP_PORT}:9080'

Reply:
fixed